docs/Monitored_Only_ZFS.md

```diff
@@ -0,0 +1,208 @@
+       # password is "vagrant"
+       vagrant sh -c 'cd /tmp; sudo tar xzvf lustre-zfs-el7-installer.tar.gz; cd lustre-zfs; sudo ./install' mds1 mds2 oss1 oss2
+   ```
+
+
+## Installing IML:
+
+1.  Copy the IML build to the /tmp directory in your admin node:
+    ```
+    scp -P 2222 ~/Downloads/iml-4.0.0.0.tar.gz vagrant@127.0.0.1:/tmp/.
+    # password is "vagrant"
+    ```
+2. ssh into the admin box and install the build:
+    ```
+    vagrant ssh
+    [vagrant@ct7-adm ~]$ sudo su - # (or "sudo -s")
+    [vagrant@ct7-adm ~]# cd /tmp
+    [vagrant@ct7-adm ~]# tar xvf <buildname>.tar.gz
+    [vagrant@ct7-adm ~]# cd <build folder>
+    [vagrant@ct7-adm ~]# ./install --no-dbspace-check
+    ```
+3. Update the /etc/hosts file on your computer to include the following line:
+    ```
+    127.0.0.1 ct7-adm.lfs.local
+    ```
+4. Test that a connection can be made to IML by going to the following link in your browser:
+https://ct7-adm.lfs.local:8443
+
+## Adding and Configuring Servers
+You should now be able to see IML when navigating to https://ct7-adm.lfs.local:8443. Click on the login link at the top right and log in as the admin. Next, go to the server configuration page and add the following servers:
+```
+ct7-mds[1,2].lfs.local,ct7-oss[1,2].lfs.local
+# Make sure to select "Monitored Server Profile" for the servers profile
+```
+This will take some time (around 5 to 10 minutes) but all four servers should add successfully.
+There will be alters and warnings about LNET. Ignore for now.
+
+Once all servers have been added with "Monitored Server Profile", each server will need to know which interface should be assigned the lustre network.
+ssh to each server (mds1, mds2, oss1, oss2)
+vagrant ssh <server>  and as root (sudo su ) run the following commands:
+```
+    systemctl stop firewalld ; systemctl disable firewalld 
+    systemctl stop NetworkManager; systemctl disable NetworkManager 
+    systemctl start ntpd 
+    echo 'options lnet networks=tcp0(enp0s9)' > /etc/modprobe.d/lustre.conf 
+    modprobe lnet 
+    lctl network configure 
+    /sbin/modprobe zfs
+    genhostid
+```
+
+The IML GUI should show that the LNET and NID Configuration is updated (IP Address 10.73.20.x to use `Lustre Network 0`). All alets are cleared.
+
+
+## Creating a monitored only zfs based Lustre filesystem
+
+The lustre filesystem will be created from the command line on zpools and IML GUI will be used to scan for the filesytem.
+Note that VM Disks (ata-VBOX_HARDDISK...) will be mapped as /dev/sd devices. Use "ls -l /dev/disk/by-id/" to list the VM Disks available. 
+
+- Management Target:
+```
+    vagrant ssh mds1
+    sudo -s
+    #note use /dev/sdb (512M) for the mgt
+    zpool create mgs -o cachefile=none /dev/disk/by-id/ata-VBOX_HARDDISK_VBe85051d4-e6ae953d -f
+    mkfs.lustre --reformat --failover 10.73.20.12@tcp --mgs --backfstype=zfs mgs/mgt
```

Comment:
Should be able to use host names here also, instead of IP addresses.

Reply:
fixed